utils/src/stages/stage.ts

```diff
@@ -79,7 +87,8 @@ export interface StageProgressConfig {
 
 export type StageConfig =
   | ChatStageConfig
-  | ElectionStageConfig
+  | ItemRankingStage
```

Comment:
FYI: Use `RankingStageConfig`, not `ItemRankingStage | ParticipantRankingStage`. Fixed in new PR.